src/mlpack/core/data/tokenizer/char_split.hpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file char_split.hpp
+ * @author Jeffin Sam
+ *
+ * Implementation of CharSplit class which tokenizes using single character.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_DATA_CHAR_SPLIT_HPP
+#define MLPACK_CORE_DATA_CHAR_SPLIT_HPP
+
+#include <mlpack/prereqs.hpp>
+#include "mlpack/core/boost_backport/boost_backport_string_view.hpp"
+
+namespace mlpack {
+namespace data {
+/**
+ * A simple CharSplit class.
+ *
+ * The class is used to split the documents using a set of characters, that
```

Comment:
Looks like the description is written for `SplitByChar`.

Reply:
Changed it accordingly